elements/lisk-tree/src/merkle_tree/calculate.ts

```diff
@@ -0,0 +1,183 @@
+import { hash } from '../../../lisk-cryptography/dist-node';
+import { LEAF_PREFIX, NODE_HASH_SIZE } from './constants';
```

Comment:
why is there an entire new `calculate` file? we can put this in `utils`

Reply:
I think it was Shu's and Alessandro's idea but don't remember the details now